spatialprofilingtoolbox/workflow/assets/.workflow.config.template

```diff
@@ -1,8 +1,9 @@
 [general]
 db_config_file = path/to/db.config
 executor = local
-sif_file = path/to/sif/file
 excluded_host = None
+orchestrator = local
+image = name/of/or/path/to/docker/or/singularity/image/if/not/local
```

Comment:
The template text here suggests that a more descriptive name than "image" might be desired.
How about:
`image_name_or_path`